Looks like we need to figure out a different way to install terraform
```
#17 1.161 E: Unable to locate package terraform
#17 ERROR: executor failed running [/bin/sh -c apt-get install terraform -y]: exit code: 100
------
 > [gardener-extension-gcve-tm-run 6/7] RUN apt-get install terraform -y:
------
executor failed running [/bin/sh -c apt-get install terraform -y]: exit code: 100
Traceback (most recent call last):
  File "<string>", line 182, in <module>
  File "/usr/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '('docker', '--config', '/tmp/tmp7rey_vfz', 'build', '--progress', 'plain', '--target', 'gardener-extension-gcve-tm-run', '--tag', 'eu.gcr.io/gardener-project/gardener/extensions/vsphere-gcve-tm-run:v0.19.0-dev-97ee8623a4237a5b6daec7812c6d01eefcfa883d', '--file', 'main-image/Dockerfile', 'main-image/')' returned non-zero exit status 1.

```

--- a/Dockerfile
+++ b/Dockerfile
@@ -34,5 +34,5 @@ COPY --from=builder /go/bin/gcve-setup /gcve-setup
 RUN apt-get update && apt-get install bash wget curl unzip software-properties-common gnupg2 -y
 RUN curl -fsSL https://apt.releases.hashicorp.com/gpg | apt-key add -
 RUN apt-add-repository "deb [arch=$(dpkg --print-architecture)] https://apt.releases.hashicorp.com $(lsb_release -cs) main"
-RUN apt-get install terraform -y
+RUN apt-get update && apt-get install terraform -y
 RUN curl -sSL https://sdk.cloud.google.com | bash

